multi_physics/QED/python_bindings/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+set(name pxr_qed)
+
+set(_PY_DEV_MODULE Development.Module)
+if(CMAKE_VERSION VERSION_LESS 3.18.0)
+    # over-specification needed for CMake<3.18
+    #   https://pybind11.readthedocs.io/en/latest/compiling.html#findpython-mode
+    #   https://cmake.org/cmake/help/v3.18/module/FindPython.html
+    set(_PY_DEV_MODULE Development)
+endif()
+find_package(Python COMPONENTS Interpreter ${_PY_DEV_MODULE} REQUIRED)
```

Comment:
I realized that this part requires a modern CMake (3.12+).

The proper/ideal treatment of pybind11 was only added in 2.6+.
If 2.4+ builds for you then go with that, I just personally dropped it because of  the modernized CMake logic.

Not that you don't need to install an old pybind11 with apt, just get it from `pip` on the fly :)

Reply:
I followed your suggestion: now I use `pip` to install `pybind11`